target/linux/mvebu/image/Makefile

```diff
@@ -25,22 +25,17 @@ endef
 # Partition 1:   32768 sectors
 # Partition 2:   98304 sectors (configurable)
 
-define Build/boot-scr-cfpro
-	rm -f $@.bootscript
-	mkimage -A arm -O linux -T script -C none -a 0 -e 0 -d cfpro-boot.script $@.bootscript
-endef
-
-define Build/boot-scr-cfbase
-	rm -f $@.bootscript
-	mkimage -A arm -O linux -T script -C none -a 0 -e 0 -d cfbase-boot.script $@.bootscript
+define Build/boot-scr
+	rm -f $@-boot.scr
+	mkimage -A arm -O linux -T script -C none -a 0 -e 0 -d $(DEVICE_NAME).bootscript $@-boot.scr
 endef
 
 define Build/boot-img
 	rm -f $@.boot
 	mkfs.fat -C $@.boot 16384
 	$(foreach dts,$(DEVICE_DTS), mcopy -i $@.boot $(DTS_DIR)/$(dts).dtb ::$(dts).dtb;)
 	mcopy -i $@.boot $(IMAGE_KERNEL) ::zImage
-	mcopy -i $@.boot $@.bootscript ::boot.scr
+	-mcopy -i $@.boot $@-boot.scr ::boot.scr
```

Comment:
Why is this allowed to fail now?

Reply:
Seems like not every device uses `boot.scr` file, look at PR #684. That way the `boot-scr-dummy` define in mentioned PR won't be necessary.